test/case_event_counts.ipynb

```diff
@@ -0,0 +1,3486 @@
+{
```

Comment:
<p>wow a lot of imports!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pedrito87/event_count_case/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>yeah</p>